src/entities/RfqmTransactionSubmissionEntity.ts

```diff
@@ -68,7 +70,7 @@ export class RfqmTransactionSubmissionEntity {
         this.metadata = opts.metadata || null;
         this.nonce = opts.nonce || null;
         this.orderHash = opts.orderHash;
-        this.status = opts.status || RfqmTranasctionSubmissionStatus.Submitted;
+        this.status = opts.status || RfqmTransactionSubmissionStatus.Submitted;
```

Comment:
Why do we want to add a default here? Can we just be explicit about what status we set?

Reply:
That's out of scope for this PR. I only corrected spelling of the enum from `RfqmTranasctionSubmissionStatus` to `RfqmTransactionSubmissionStatus`